integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/HiveContainer.java

```diff
@@ -50,8 +56,30 @@ protected void setupContainer() {
 
   @Override
   public void start() {
-    super.start();
-    Preconditions.check("Hive container startup failed!", checkContainerStatus(5));
+    try {
+      super.start();
+      Preconditions.check("Hive container startup failed!", checkContainerStatus(5));
+    } finally {
+      outputContainerServiceLog();
+    }
+  }
+
+  @Override
+  protected void outputContainerServiceLog() {
+    LOG.info(
+        "--------------------Start the Hive server information--------------------------\n{}",
+        getServiceLog(HIVE_LOG_PATH));
+    LOG.info("--------------------End the Hive server information----------------------------");
+
+    LOG.info(
+        "----------------------------Start the HDFS namenode information -----------------\n{}",
+        getServiceLog(HDFS_NAME_NODE_LOG_PATH));
+    LOG.info("----------------------------End the HDFS namenode information -------------------");
+
+    LOG.info(
+        "----------------------------Start the HDFS datanode information -----------------\n{}",
+        getServiceLog(HDFS_DATA_NODE_LOG_PATH));
+    LOG.info("----------------------------End the HDFS datanode information -------------------");
```

Comment:
BTW, for current implementation, the logs of different processes in the same container are also mixed in one log file.

Reply:
> the logs of different processes in the same container

Yes, some services like `Hive` will start several instances like `hive`, `namenode` and `datanode`

Reply:
For the different processes, if they started the multiple hive containers, how do you distinguish which container?

Reply:
@jerryshao 

When we execute the `gradle test` it will launch several Java instances to run the tests in different modules, for example, ITs in `catalog-hive` and ITs in `catalog-icebergs` will be executed in different JVM, and the log will be located in the corresponding integration test log directory. 

It is unable to handle scenarios where two or more Hive containers start at the same time in one JVM and can't determine which one is okay or not if failures occur.

Reply:
Please look at the word "different processed" I mentioned above.